src/app/components/source-graphs/source-graphs.component.ts

```diff
@@ -54,9 +55,15 @@ export class SourceGraphsComponent implements OnInit {
           ? event.clientX - this.tooltip.mid
           : document.body.clientWidth - this.tooltip.width
 
-      this.tooltipY = event.clientY
+      this.tooltipY =
+        event.clientY < document.body.clientHeight - this.tooltip.height
+          ? event.clientY
+          : document.body.clientHeight -
+            this.tooltip.height -
+            this.tooltipOffsetY
       this.lineX = event.layerX
       this.tooltipShow = 1
+      this.lineX = event.layerX + 8
```

Comment:
duplicate of https://github.com/RADAR-CNS/RADAR-Dashboard/pull/254/files#diff-07b51894df76f210fb2a7259f865741eR64